samples/bluetooth/mesh/light_ctrl/src/model_handler.c

```diff
@@ -242,11 +260,21 @@ static struct bt_mesh_elem elements[] = {
 	BT_MESH_ELEM(1,
 		     BT_MESH_MODEL_LIST(
 			     BT_MESH_MODEL_CFG_SRV,
-			     BT_MESH_MODEL_HEALTH_SRV(&health_srv, &health_pub),
+#if defined(CONFIG_BT_MESH_LARGE_COMP_DATA_SRV)
+			     BT_MESH_MODEL_HEALTH_SRV(
+					 &health_srv, &health_pub, health_srv_metadata),
+			     BT_MESH_MODEL_LIGHTNESS_SRV(
+					 &my_ctx.lightness_srv, lightness_srv_metadata),
+			     BT_MESH_MODEL_SENSOR_SRV(&sensor_srv, NULL),
```

Comment:
If there is no metadata for the model, it isn't had to be passed in macro. So sensor server can be moved out of ifdef.